plugins/modules/sefcontext.py

```diff
@@ -28,85 +27,82 @@
 options:
   target:
     description:
-    - Target path (expression).
+      - Target path (expression).
     type: str
     required: true
-    aliases: [ path ]
+    aliases: [path]
   ftype:
     description:
-    - The file type that should have SELinux contexts applied.
-    - "The following file type options are available:"
-    - V(a) for all files,
-    - V(b) for block devices,
-    - V(c) for character devices,
-    - V(d) for directories,
-    - V(f) for regular files,
-    - V(l) for symbolic links,
-    - V(p) for named pipes,
-    - V(s) for socket files.
+      - The file type that should have SELinux contexts applied.
+      - The following file type options are available:.
+      - V(a) for all files,.
+      - V(b) for block devices,.
+      - V(c) for character devices,.
+      - V(d) for directories,.
+      - V(f) for regular files,.
+      - V(l) for symbolic links,.
+      - V(p) for named pipes,.
+      - V(s) for socket files.
```

Comment:
A trailing comma should also be left alone:
```suggestion
      - The following file type options are available:
      - V(a) for all files,
      - V(b) for block devices,
      - V(c) for character devices,
      - V(d) for directories,
      - V(f) for regular files,
      - V(l) for symbolic links,
      - V(p) for named pipes,
      - V(s) for socket files.
```